pkg/controller/storagecluster/generate.go

```diff
@@ -41,3 +41,11 @@ func generateNameForCephFilesystemSC(initData *ocsv1.StorageCluster) string {
 func generateNameForCephBlockPoolSC(initData *ocsv1.StorageCluster) string {
 	return fmt.Sprintf("%s-ceph-rbd", initData.Name)
 }
+
+func generateNameForRBDSnapshotClass(initData *ocsv1.StorageCluster) string {
+	return fmt.Sprintf("%s-rbdplugin-snapclass", initData.Name)
```

Comment:
%s/snapclass/snapshotclass

Reply:
Hi @Madhu-1 , I have created the name in accordance with the following yaml file,
https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/rbd/snapshotclass.yaml

One more question is, in the below comment, where @humblec asks for a common functionality, but in the above string `%s/snapclass/snapshotclass`, we don't have a provision to show whether it is created from `rbd` or `cephfs`.